myapp/src/pages/LoginPage.js

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import BeatLoader from "react-spinners/ClipLoader";
+
+import InputField from "../components/InputField";
+
+function LoginPage() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const [usernameError, setUsernameError] = useState("");
+  const [passwordError, setPassswordError] = useState("");
+
+  let [loading, setLoading] = useState(false);
+
+  const navigate = useNavigate();
+
+  function setData(event) {
```

Comment:
```suggestion
  function login(event) {
```
We should always try to name our variables/functions with meaningful names. `setData` is true as the function is setting the data, but the ultimate outcome of it is to login. We should keep that in mind when naming.

Reply:
Changed it.